resources/js/components/ui/calendar/FullCalendar.tsx

```diff
@@ -0,0 +1,266 @@
+            ]);
+          }
+        }
+      }
+    }),
+  );
+
+  /** Returns the UI text describing the selected days range. */
+  function getDaysSelectionText() {
+    const m = mode();
+    switch (m) {
+      case "month":
+        return daysSelection().start.toLocaleString({month: "long", year: "numeric"});
+      case "week": {
+        const fitsInMonth = daysSelection().start.hasSame(daysSelection().end, "month");
+        return `${daysSelection().start.toLocaleString({
+          day: "numeric",
+          month: fitsInMonth ? undefined : "long",
+        })} — ${daysSelection().end.toLocaleString({day: "numeric", month: "long", year: "numeric"})}`;
+      }
+      case "day":
+        return daysSelection().start.toLocaleString({weekday: "long", day: "numeric", month: "long", year: "numeric"});
+      default:
+        return m satisfies never;
+    }
+  }
+
+  return (
+    <div {...htmlAttributes.merge(divProps, {class: "flex items-stretch gap-1"})}>
+      <div class="flex flex-col items-stretch gap-1" style={{"flex-basis": "min-content"}}>
+        <TinyCalendar
+          locale={props.locale}
+          showWeekdayNames={true}
+          holidays={props.holidays}
+          selection={daysSelection()}
+          month={tinyCalMonth()}
+          setMonth={setTinyCalMonth}
+          getHoverRange={getRange}
+          onDayClick={(day, range) => {
+            setTinyCalMonth(day);
+            setDaysSelection(range!);
+          }}
+          onDayDoubleClick={(day) => {
+            // Switch between day and week modes.
+            setMode(mode() === "day" ? "week" : "day");
+            setDaysSelectionAndMonthFromDay(day);
+          }}
+          onMonthNameClick={() => {
+            setMode("month");
+            setDaysSelection(getRange(tinyCalMonth()));
+          }}
+          setVisibleRange={setTinyCalVisibleRange}
+        />
+        <ResourcesSelector
+          class="overflow-y-auto"
+          resourceGroups={props.resourceGroups}
+          mode={resourcesSelectionMode()}
+          selection={selectedResources()}
+          setSelection={setSelectedResources}
+        />
+      </div>
+      <div class="grow flex flex-col items-stretch gap-1">
+        <div class="pt-1 pr-1 flex items-stretch gap-1">
+          <div>
+            <Button class="h-full secondarySmall !rounded-r-none" onClick={() => moveDaysSelection(-1)}>
+              <IoArrowBackOutline class="text-current" />
+            </Button>
+            <Button
+              class="h-full secondarySmall !rounded-l-none"
+              style={{"margin-left": "-1px"}}
+              onClick={() => moveDaysSelection(1)}
+            >
+              <IoArrowForwardOutline class="text-current" />
+            </Button>
+          </div>
+          <Button class="secondarySmall" onClick={() => setDaysSelectionAndMonthFromDay(currentDate())}>
+            <Capitalize text={t("calendar.today")} />
```

Comment:
To teraz są dwa przyciski podobne do "Dzisiaj". Jeden ten w TinyCalendar co czasem skacze do dzisiaj, ale nie zaznacza, i jeden tutaj który zaznacza. Wiem że byłeś dumny z tego "idź w inne interesujące miejsce", ale jako że nie tylko ja narzekałem, to rozważ czy tego nie uprościć jednak. :) Ja bym zrobił jeden przycisk w TinyCalendar który zawsze skacze do dzisiaj i wybiera dzisiaj.